package.json

```diff
@@ -33,9 +33,10 @@
     "@lion/tooltip": "^0.21.3",
     "@lion/validate-messages": "^0.7.4",
     "@open-wc/scoped-elements": "^2.0.0-next.4",
-    "dark-theme-utils": "^0.3.2"
+    "dark-theme-utils": "^0.3.2",
+    "lit": "2.0.0-rc.4"
   },
   "devDependencies": {
     "bkl": "latest"
   }
-}
+}
```

Comment:
nitpick: prettier didn't run? anyhow I approved your PR of course :)

Reply:
Yeah prettier in bkl in browser is very weird atm, gives way different results vs local vscode editor, very annoying. Need to fix ASAP maybe even make commando, not sure.

Reply:
For what it worth ... saving one file vs saving all does not do the same formatting.

Formatting all is doing the correct behaviour IIRC